docs/adding_packages/build_and_package.md

```diff
@@ -0,0 +1,36 @@
+# Build and Package
+
+This document gathers all the relevant information regarding the general lines to follow while writing either the `build()` or the `package()` methods.
+Both of these methods often take advantage of build helpers build binaries and install to the `package_folder`.
+
+<!-- toc -->
+## Contents
+
+  * [Build Method](#build-method)
+  * [Package](#package)<!-- endToc -->
+
+## Build Method
+
+* `build()` method should focus on build only, not installation. During the build, nothing should be written in `package` folder. Installation step should only occur in `package()` method.
+
+* The build itself should only rely on local files. Nothing should be downloaded from internet during this step. If external files are required, they should come from `requirements` or `build_requirements`, in addition to source files downloaded in `source()` or coming from recipe itself.
```

Comment:
```suggestion
* The build itself should only rely on local files. Nothing should be downloaded from internet during this step. If external files are required, they should come from `requirements` or `tool_requirements`, in addition to source files downloaded in `source()` or coming from recipe itself.
```

Reply:
That's consufising the method is called `build_requirements` which is what sentence is referring too I think

Reply:
I suppose we need to avoid confusion and use consistent naming/terminology across the docs. either tool_requires everywhere, or build_requires everywhere.